src/test/java/ConversionUtil.java

```diff
@@ -0,0 +1,20 @@
+import com.owlike.genson.Genson;
+import org.apache.http.HttpEntity;
+import org.apache.http.util.EntityUtils;
+
+import javax.swing.text.html.parser.Entity;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class ConversionUtil {
+    private Genson genson;
+    static private ConversionUtil instance;
+    static public ConversionUtil getConversionUtil(){if(instance == null){instance = new ConversionUtil();} return instance;}
+    public ConversionUtil(){
```

Comment:
You should make the constructor `private`. Else it defeats the purpose of a singleton as I can now do `new ConversionUtil()` and get a new instance.